documentation/tutorials/buildlinux.rst

```diff
@@ -0,0 +1,58 @@
+=================================
+  How to build the Linux kernel
+=================================
+
+.. contents::
+   :depth: 1
+   :local:
+   :backlinks: none
+
+.. highlight:: console
+
+Get a Linux repository
+----------------------
+After successfully installing kw, it's time to build the Linux kernel using it.
+Let's start by cloning the Linux repository from Torvalds into our current
+working directory::
+
+  git clone https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git
+
+Then, change the current directory to the Linux repository::
+
+  cd linux
+
+Get .config
+-----------
+To build the kernel, a `.config` file is necessary. This file contains
+configurations about the kernel compilation process.
+To retrieve this file from your own computer, run, in Debian::
+
+  cp /boot/config-`uname -r` .config
+
+Or in Arch::
+
+  zcat /proc/config.gz > .config
+
+Change .config a bit
+--------------------
+A small modification you can make in the `.config` file is changing the kernel
+release name. To do that, run::
+
+  kw build --menu
+
+The default menu config option is set in the kworkflow.config file as nconfig,
+but it can be changed, for instance, to menuconfig, or whichever you prefer.
+After running the command above, go to `General setup`, then `Local version -
+append to kernel release`, choose any name you like, save the new configuration
+and exit the menu.
+
+.. note::
+    You can read more about the `kworkflow.config` file in our manual.
+
+Build!
+------
+Building the kernel now is as easy as invoking::
+
+  kw build
```

Comment:
Also comment about the short version: `kw b`

Reply:
Add a tiny explanation about the `-jNUMBER` flag. I think it is important to highlight that this command will handle this for users.